collagraph/collagraph.py

```diff
@@ -33,6 +36,15 @@ def create_element(type, props=None, *children) -> VNode:
     return VNode(type, reactive(props or {}), children or tuple(), key)
 
 
+def render_slot(name, props, slots):
```

Comment:
I figured that there should be `render_slot` function as well for use in functional components.

Reply:
Do you have tests for this?

Reply:
Yes, although not 100%. Now that I think about it: I could actually call this method from within `Component` in order to re-use the code. Might make more sense than adding tests.